integration/test_aggregation_with_redis.py

```diff
@@ -0,0 +1,1853 @@
+import asyncio
```

Comment:
Do we really have to duplicate `integration/test_aggregation_integration.py`? Can't we just parameterize the suite?

Reply:
Done